style.css

```diff
@@ -1,123 +1,103 @@
-*{
+* {
 	box-sizing: border-box; 
 }
-html{
+
+html {
 	font-size: 20px; 
 }
+
 body {
 	background-color:#771668;
 	width: 100%;
-	margin-top:80px;
-	margin-left:0px;
-	margin-right:0px;
-	margin-bottom:0px;
+	margin-top: 200px;
+	margin-left: 0px;
+	margin-right: 0px;
+	margin-bottom: 0px;
+}
+
+a, p, h1, h2, h3, h4, h5, h6 {
+	color: white;
 }
 
-header{
+header {
 	position: fixed;
-	top:0px;
-	left:0px;
+	top: 0px;
+	left: 0px;
 	background-color: #2d094f;
-	width:100%;
+	width: 100%;
 	padding: 10px;
-	border:3px solid black;
-	z-index:2;
+	border: 3px solid black;
+	z-index: 1;
 }
 
-.nav {
+.id-section-bar {
 	font-size: 1.5rem;
 	color: white;
 	display: flex;
-	gap:40px;
-	text-decoration: none;
-			
-}
-section.intro{
-		display: flex;
-		align-items: center;
-		flex-direction: column;
-		width: 100%;
-		margin-bottom:250px;
-		gap: 18px;			
+	gap: 40px;
+	text-decoration: none;		
 }
 
-p.intro{
-	color: white;
-	font-size: 6rem;
+section.intro {
 	display: flex;
-	justify-content: center;
-	flex-wrap: wrap;
-	margin:0px;
 	align-items: center;
-	gap:0px;
+	flex-direction: column;
+	width: 100%;
+	margin-bottom: 500px;	
 }
 
-p.little-intro{
-	color: white;
-	font-size: 2rem;
-	display: flex;
-	justify-content: center;
+p.intro {
+	font-size: 6rem;
+	margin: 0px;
 }
 
-div.about-catcher{
-    width: 100%;
- 	padding: 10px;
-    background-color: #3C0051;
+p.second-intro {
+	font-size: 2rem;
 }
 
-section.about{
-	    width: 100%;
-	    height: 800px;
-	    background-color: #3C0051;
-	    margin-top: 30px;
-	    display: flex;
-	    flex-wrap: wrap;
-	    flex-direction: column;
 
+div.about-piece{
+	width: 100%;
+	background-color: #3C0051;
+	margin-top: 30px;
+	border: 5px solid black;
+	padding: 50px;
 }
 
-.about-img{
-	width:600px;
-	height:600px;
-	object-fit:contain;	
-	margin-right: 10px;
+.about-contents {
+	display: flex;
+    flex-wrap: nowrap;
+    flex-direction: column;
+    align-content: stretch;
+    align-items: center;
 }
 
-h2.about{
-	margin-left:15px;
-	color: white;
-	font-size: 2rem;
-	text-decoration: underline;	
+p.about-contents {
+	font-size: 1rem;
 }
 
-p.about{
-	width:60%;
-	color: white;
-	font-size: 1.5rem;
-	margin-left:15px;
-}
 
-div.portfolio-catcher {
-    width: 100%;
- 	padding: 10px;
-    background-color: #453952;
-	margin-top:50px;
+img.about-image{
+	object-fit: contain ;
+	width: 70%;
+	height: 500px;
+	display: flex;
 }
 
-section.portfolio-intro { width: 100%;
+section.portfolio-intro { 
+	margin-top: 100px;
+	width: 100%;
     height: 150px;
```

Comment:
height here should not be set as per [previous comment ](https://github.com/Joshsilas/JoshPortfolio/pull/5/files/2134f6a6608213840c83bd6b6d292ed4b4dbaae2..1fd9410b46c06b60bae8776114d4d9ea9f202d1e?diff=split&w=0#r1261384809)